static/usr/lib/systemd/system/run-mnt-data.mount.d/late-umount.conf

```diff
@@ -0,0 +1,2 @@
+[Unit]
+DefaultDependencies=no
```

Comment:
Is this to make sure these unmount units survive umount target?

Reply:
Yes, I thought I commented it. But I did not. I will add a comment.

Reply:
Commented